lib/ts/recipeImplementation.ts

```diff
@@ -308,6 +308,27 @@ export default function RecipeImplementation(recipeImplInput: {
                     return domain === normalisedsessionDomain;
                 }
             }
+        },
+
+        getClockSkewInMillis: function ({
+            accessTokenPayload
+        }: {
+            accessTokenPayload: any;
+            responseHeaders: Headers;
+        }): number {
+            logDebugMessage("getClockSkewInMillis: called");
+
+            const tokenIssuedAt = accessTokenPayload?.iat;
+            if (tokenIssuedAt === undefined || typeof tokenIssuedAt !== "number") {
+                logDebugMessage("getClockSkewInMillis: payload iat is undefined or not a number, returning 0");
+                return 0;
+            }
```

Comment:
This should never happen, so we can throw an error here. (That way test failures will let us know if this is something we do need to worry about)
This information should be present in all access tokens I think. (maybe under a different name for older version)